### PR TITLE
Added .editorconfig

### PR DESCRIPTION
**Pull request by nlok5923**
fixes : #143

### CODE REVIEW

**Comment by nlok5923** [comment]
@iiitv/project-maintainers please review this pr.

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+[*]
+indent_style = space
+indent_size = 4
+insert_final_newline = true
+trim_trailing_whitespace = true
+end_of_line = lf
+charset = utf-8
+
+[*.py]
+max_line_length = 119
```
Comment (aashutoshrathi) [inline .editorconfig:12]: The things you added in `[*]` are meant to be only for `[*.py]` right?
If so, better add them in `[*.py]`

**Review by aashutoshrathi — CHANGES_REQUESTED** [review]
Just recheck and add only common things in `[*]`

**Review by aashutoshrathi — APPROVED** [review]
LGTM

**Review by anshumanv — APPROVED** [review]

**Comment by AmanRaj1608** [comment]
LGTM, merging

**Pull request merged**